### PR TITLE
Setup of CI/CD

### LINKED ISSUE #2: Setup circle-ci/travis for CI

### PR DESCRIPTION
**Pull request by fenilgandhi**
**Updated**
This PR add the following features into the codebase.
- Adds Circle-CI integration to the project.
- Adds Heroku setup file for deployment after tests have passed.
- Adds Lint testing.
- Updates test parameter in package.json to exit tests are running.

### CODE REVIEW

**Review by nitishmittal1990 — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ## Uncomment the following lines to enable mongo-db
+    #  - image: circleci/mongo:4.0.1
+    # environment:
+    #   - DB_URL:mongodb://127.0.0.1:27017
+    working_directory: ~/batch-12-WeConnect
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@latest'
```
Comment (rishabh1403) [inline .circleci/config.yml]: you don't need to update npm. It comes bundled with node

**Review by rishabh1403 — CHANGES_REQUESTED** [review]
There's no deployment here. All the circle ci is doing is installing node modules.
Also, some files are missing new line at end. Please resolve those

File: .circleci/config.yml (hunk 1 above)
Comment (fenilgandhi) [reply .circleci/config.yml]: Ok. It was only a precaution.

Comment (fenilgandhi) [reply .circleci/config.yml]: yes, there are no deployment files and it's intentional because the deployments are auto-managed using app integrations.

**Comment by nimish-gupta** [comment]
> This PR add the following features into the codebase.
> 
>     * Updates templates for creating new issues and prs on github
> 
>     * Adds WTFPL 2.0 LICENSE to the project.
> 
>     * Add Circle-CI integration to the project.

All the above point mentioned should be done in a another PR, PR should always (most of the time) should contain of single feature.

**Comment by fenilgandhi** [comment]
@nimish-gupta I have done the required changes.

File: .circleci/config.yml (hunk 1 above)
Comment (rishabh1403) [reply .circleci/config.yml]: The point is to learn how to do those using CI/CD
Example
- test if linting is correct
- run tests
- build code ( if applicable)
- deploy


Unless all these are present in the circle ci, the file only with installing node modules isn't doing anything
cc @nimish-gupta

**Review by nimish-gupta — CHANGES_REQUESTED** [review]
Hey please rebase it with the latest master and then ask for review, because in this PR there might be a change in max 3-4 files (one config, and other tweaking in some tests).

File: .circleci/config.yml (hunk 1 above)
Comment (fenilgandhi) [reply .circleci/config.yml]: Ok, i have made all of the suggested changes. @rishabh1403

**Comment by rishabh1403** [comment]
frontend deployment missing

**Comment by fenilgandhi** [comment]
Ok, i have added the frontend deployment to CI too.

**Review by rishabh1403 — APPROVED** [review]

**Review by vineetvk01 — APPROVED** [review]
LGTM

**Pull request merged**